meadow/src/Gist.hs

```diff
@@ -13,49 +13,52 @@
 {-# LANGUAGE TypeOperators         #-}
 
 module Gist
-  ( API
-  , GistAPI
-  , getGists
-  , createNewGist
-  , getGist
-  , updateGist
-  , Owner(..)
-  , GistId(..)
-  , Gist(..)
-  , GistFile(..)
-  , NewGist(..)
-  , NewGistFile(..)
-  ) where
+    ( API
+    , GistAPI
+    , getGists
+    , createNewGist
+    , getGist
+    , updateGist
+    , Owner(..)
+    , GistId(..)
+    , Gist(..)
+    , GistFile(..)
+    , NewGist(..)
+    , NewGistFile(..)
+    ) where
 
```

Comment:
I'm also gonna push back here. I think it's time to create a common server project. There's no good reason to have two different `Gist` modules in the backend code.

Reply:
done. should `interpreter` be renamed to `playground-common` or something?

Reply:
Yeah. Assuming `meadow` will eventually become `marlow-playground-server` (or similar) then I think `playground-common` is a good choice. 👍

Reply:
done